packages/expo-image-picker/ios/EXImagePicker/EXImagePickerCameraRequester.m

```diff
@@ -0,0 +1,54 @@
+// Copyright 2016-present 650 Industries. All rights reserved.
+
+#import <EXImagePicker/EXImagePickerCameraRequester.h>
+#import <UMCore/UMDefines.h>
+#import <UMPermissionsInterface/UMPermissionsInterface.h>
+
+#import <AVFoundation/AVFoundation.h>
+
+
+@implementation EXImagePickerCameraRequester
+
++ (NSString *)permissionType {
+  return @"camera";
+}
+
+- (NSDictionary *)getPermissions
+{
+  AVAuthorizationStatus systemStatus;
+  UMPermissionStatus status;
+  NSString *cameraUsageDescription = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"NSCameraUsageDescription"];
+  NSString *microphoneUsageDescription = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"NSMicrophoneUsageDescription"];
+  if (!(cameraUsageDescription && microphoneUsageDescription)) {
+    UMFatal(UMErrorWithMessage(@"This app is missing either NSCameraUsageDescription or NSMicrophoneUsageDescription, so audio/video services will fail. Add one of these keys to your bundle's Info.plist."));
```

Comment:
```suggestion
    UMFatal(UMErrorWithMessage(@"This app is missing either 'NSCameraUsageDescription' or 'NSMicrophoneUsageDescription', so audio/video services will fail. Ensure both of these keys exist in your bundle's (does Ingo.plist belong to bundle? 🤔) Info.plist."));
```
edit: https://docs.expo.io/versions/latest/workflow/configuration/#ios <- there is `infoPlist` key 😅 

```suggestion
UMFatal(UMErrorWithMessage(@"This app is missing either 'NSCameraUsageDescription' or 'NSMicrophoneUsageDescription', so audio/video services will fail. Ensure both of these keys exist in app's Info.plist."));
```